tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -189,7 +193,10 @@ func runBugReportCommand(_ *cobra.Command, logOpts *log.Options) error {
 		if tempDir != "" {
 			archiveDir = tempDir
 		}
-		if err := archive.Create(archiveDir, outPath); err != nil {
+		curTime = time.Now()
+		err := archive.Create(archiveDir, outPath)
+		common.LogAndPrintf("Time used for creating the tar file is %v.", time.Since(curTime))
```

Comment:
do we want LogAndPrintf? seems like just log is sufficient.

If so, we need to add explicit `\n` to the end

Reply:
Not sure generally where log goes. 

If log only prints to `bug-report.go`, we probably have to change it to only print because after we generate archive, we cannot add any log to `bug-report.go`.

Reply:
log goes to the bug-report.tar.gz, LogAndPrintf goes to that and also the output of `istioctl bug-report`.

Reply:
Good point about it being too late